dlt/destinations/impl/athena/athena.py

```diff
@@ -389,6 +389,9 @@ def __init__(
             config,
             capabilities,
         )
+        SupportsStagingDestination.__init__(
```

Comment:
In athena this method actually is implemented because we need special behavior for athena to work properly. I think here we can just ignore this setting and mention it in the docs.